tests/queries/0_stateless/03208_keepermap_incomplete_data_drop.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS 03208_keepermap_test SYNC;
+
+CREATE TABLE 03208_keepermap_test (key UInt64, value UInt64) Engine=KeeperMap('/' || currentDatabase() || '/test03208') PRIMARY KEY(key);
+INSERT INTO 03208_keepermap_test VALUES (1, 11);
+SYSTEM ENABLE FAILPOINT keepermap_fail_drop_data;
+DROP TABLE 03208_keepermap_test;
```

Comment:
Shouldn't that `DROP TABLE 03208_keepermap_test` throw an exception?
(because of `SYSTEM ENABLE FAILPOINT keepermap_fail_drop_data`)

Reply:
yeah you are right, `database_atomic_wait_for_drop_and_detach_synchronously` needs to be `0`
locally I didn't have SYNC drops turned on so I didn't notice it

the reason we didn't get exceptions was because the failpoint is defined with `ONCE` so the drop will only fail the first time and succeed on the subsequent retries

Reply:
Maybe it makes sense to call `DROP TABLE SYNC` and expect an exception in the test, WDYT?
Just in order to make the test more related to the case.